tests/testthat/test-iSEEnrich.R

```diff
@@ -3,11 +3,18 @@ test_that("test iSEEnrich", {
   GO_collection <- "GO"
   Mm_organism <- "org.Mm.eg.db"
   gene_id <- "SYMBOL"
+  clusters <- "Primary.Type"
+  groups <- "Secondary.Type"
+  reddim_type <- "PCA"
```

Comment:
just a pointer again to note down what we can also do: Trigger the errors by selecting wrong values for such parameters - see the comment above about adding some extra checks at the beginning of the function

Reply:
[f9942cd](https://github.com/NajlaAbassi/iSEEfier/pull/10/commits/f9942cd896004c53b39ae967eeeeffc7c849db47)